crates/gen/src/types/delegate.rs

```diff
@@ -130,9 +130,7 @@ impl Delegate {
                 #phantoms
             }
             unsafe impl<#constraints> ::winrt::RuntimeType for #name {
-                fn signature() -> String {
-                    #signature
-                }
+                const SIGNATURE: ::winrt::ConstBuffer = { #signature };
```

Comment:
And here.

Reply:
The braces are necessary here because the signature is calculated through several lines of const code.